src/Administration/Framework/Twig/ViteFileAccessorDecorator.php

```diff
@@ -21,47 +22,68 @@ class ViteFileAccessorDecorator extends FileAccessor
      * @internal
      */
     public function __construct(
-        private readonly array $configs,
-        private readonly FilesystemOperator $filesystem,
-        private readonly Packages $packages,
+        array $configs,
+        private readonly \Symfony\Component\Asset\Package $package,
```

Comment:
import namespace? 🤔

Reply:
guess what we have our very own package class, but I can import as AssetPackage alias